src/equations/1d/compressible_euler.jl

```diff
@@ -0,0 +1,847 @@
+@doc raw"""
+    CompressibleEulerEquations1D
+
+The compressible Euler equations for an ideal gas in two space dimensions.
+"""
+struct CompressibleEulerEquations1D <: AbstractCompressibleEulerEquations{1, 3}
+  gamma::Float64
+end
+
+function CompressibleEulerEquations1D()
+  gamma = parameter("gamma", 1.4)
+
+  CompressibleEulerEquations1D(gamma)
+end
+
+
+get_name(::CompressibleEulerEquations1D) = "CompressibleEulerEquations1D"
+varnames_cons(::CompressibleEulerEquations1D) = @SVector ["rho", "rho_v1", "rho_e"]
+varnames_prim(::CompressibleEulerEquations1D) = @SVector ["rho", "v1", "p"]
+
+
+# Set initial conditions at physical location `x` for time `t`
+function initial_conditions_density_pulse(x, t, equation::CompressibleEulerEquations1D)
+  rho = 1 + exp(-(x[1]^2 ))/2
+  v1 = 1
+  rho_v1 = rho * v1
+  p = 1
+  rho_e = p/(equation.gamma - 1) + 1/2 * rho * v1^2
+  return @SVector [rho, rho_v1, rho_e]
+end
+
+"""
+    initial_conditions_density_wave(x, t, equation::CompressibleEulerEquations1D)
+
+test case for stability of EC fluxes from paper: https://arxiv.org/pdf/2007.09026.pdf
+domain [-1, 1]
+mesh = 4x4, polydeg = 5
+"""
+
+function initial_conditions_density_wave(x, t, equation::CompressibleEulerEquations1D)
+  v1 = 0.1
+  rho = 1 + 0.98 * sinpi(2 * (x[1] - t * v1))
+  rho_v1 = rho * v1
+  p = 20
+  rho_e = p / (equation.gamma - 1) + 1/2 * rho * v1^2
+  return @SVector [rho, rho_v1, rho_e]
+end
+#=
+function initial_conditions_pressure_pulse(x, t, equation::CompressibleEulerEquations1D)
+  rho = 1
+  v1 = 1
+  rho_v1 = rho * v1
+  p = 1 + exp(-(x[1]^2))/2
+  rho_e = p/(equation.gamma - 1) + 1/2 * rho * v1^2
+  return @SVector [rho, rho_v1, rho_e]
+end
+
+function initial_conditions_density_pressure_pulse(x, t, equation::CompressibleEulerEquations1D)
+  rho = 1 + exp(-(x[1]^2))/2
+  v1 = 1
+  rho_v1 = rho * v1
+  p = 1 + exp(-(x[1]^2))/2
+  rho_e = p/(equation.gamma - 1) + 1/2 * rho * v1^2
+  return @SVector [rho, rho_v1, rho_e]
+end
+
+function initial_conditions_constant(x, t, equation::CompressibleEulerEquations1D)
+  rho = 1.0
+  rho_v1 = 0.1
+  rho_e = 10.0
+  return @SVector [rho, rho_v1, rho_e]
+end
+=#
+function initial_conditions_convergence_test(x, t, equation::CompressibleEulerEquations1D)
+  c = 2
+  A = 0.1
+  L = 2
+  f = 1/L
+  ω = 2 * pi * f
+  ini = c + A * sin(ω * (x[1] - t))
+
+  rho = ini
+  rho_v1 = ini
+  rho_e = ini^2
+
+  return @SVector [rho, rho_v1, rho_e]
+end
+#=
```

Comment:
I think of all the commented ICs, you should keep `weak_blast_wave` and `sedov_blast_wave`